habitat-lab/habitat/datasets/rearrange/samplers/object_sampler.py

```diff
@@ -376,7 +394,12 @@ def single_sample(
 
         return new_object, target_receptacle
 
-    def set_num_samples(self):
+    def set_num_samples(self, num_objects: Tuple[int, int] = None):
+        if num_objects is not None:
+            self.num_objects = num_objects
+            assert self.num_objects[1] >= self.num_objects[0]
+        if self.num_objects is None:
```

Comment:
What will happen when the `self.num_objects` is None?

Reply:
See [here](https://github.com/facebookresearch/habitat-lab/pull/1280#discussion_r1172699552).